config/routes.rb

```diff
@@ -32,4 +32,8 @@
 
   get '/applications/new', to: 'applications#new'
   post '/applications', to: 'applications#create'
+  get '/applications/:application_id', to: 'applications#show'
+  get '/pets/:id/applications', to: 'applications#index'
+  patch '/pets/:petid/applications/:appid/approve', to: 'pets#change_adoption'
+  patch '/pets/:petid/applications/:appid/reject', to: 'pets#change_adoption'
```

Comment:
not sure if these belong here or in the pet block (feel free to move and adjust as you see fit)